app/services/testdata/StatusGeneratorFactory.scala

```diff
@@ -58,6 +58,14 @@ object StatusGeneratorFactory {
         }
       case (PHASE1_TESTS, Some(PHASE1_TESTS_COMPLETED)) => Phase1TestsCompletedStatusGenerator
       case (PHASE1_TESTS, Some(PHASE1_TESTS_RESULTS_RECEIVED)) => Phase1TestsResultsReceivedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_INVITED)) => Phase2TestsInvitedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_STARTED)) => Phase2TestsStartedStatusGenerator
+      case (PHASE2_TESTS, Some(PHASE2_TESTS_EXPIRED)) =>
+        if (generatorConfig.phase1StartTime.isDefined) {
```

Comment:
Seems strange that phase2 could be expired when phase1StartTime is not defined. I am guessing this is some form of hack just for tests though :-)